store_db.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE book_store (
+      id     INTEGER PRIMARY KEY,
```

Comment:
You can auto increment your values if you want using "SERIAL" if you want: ```  id SERIAL PRIMARY KEY, ``` . That way in each of your " INSERT INTO " will not need the id column.